shared/src/main/kotlin/com/egm/stellio/shared/model/ExpandedEntity.kt

```diff
@@ -23,18 +23,22 @@ class ExpandedEntity private constructor(
         }
     }
 
-    val id = attributes["@id"]!! as String
-    val type = (attributes["@type"]!! as List<String>)[0]
+    val id = attributes[NgsiLdParsingUtils.NGSILD_ENTITY_ID]!! as String
+    val type = (attributes[NgsiLdParsingUtils.NGSILD_ENTITY_TYPE]!! as List<String>)[0]
     val relationships by lazy { getAttributesOfType(NGSILD_RELATIONSHIP_TYPE) }
     val properties by lazy { getAttributesOfType(NGSILD_PROPERTY_TYPE) }
+    val attributesWithoutTypeAndId by lazy {
```

Comment:
- The name is not very extensible :) Better simply call it `attributes` (and rename the existing with a more appropriate name)
- Put the logic inside a function to be consistent with the other lazy properties

Reply:
Can't find an alternative name for the old attributes..

Reply:
Maybe something like `rawProperties`?

Reply:
Well, no, `properties` is confusing. `rawJsonLdProperties`?

Reply:
Fine for me